src/loadJs.js

```diff
@@ -6,15 +6,20 @@ const readFile = require('./readFile');
 
 module.exports = function loadJs(
   filepath: string,
-  options: { sync?: boolean }
+  options: { ignoreEmpty: boolean, sync?: boolean }
 ): Promise<?cosmiconfig$Result> | ?cosmiconfig$Result {
   function parseJsFile(content: ?string): ?cosmiconfig$Result {
-    if (!content) return null;
+    const isEmpty = content === '';
+    if (content == null || (isEmpty && options.ignoreEmpty)) return null;
 
-    return {
-      config: requireFromString(content, filepath),
+    const config = isEmpty ? undefined : requireFromString(content, filepath);
```

Comment:
I think this section could be improved a little. How about this:

```js
return isEmpty
  ? { config: undefined, filepath, isEmpty: true }
  : { config: requireFromString(content, filepath), filepath };
```

Reply:
just pushed